test/Microsoft.OpenApi.Tests/Models/Samples/docWithEmptyOperationSecurity.yaml

```diff
@@ -0,0 +1,20 @@
+openapi: 3.0.0
```

Comment:
Another question for you @baywet 

I introduced two new contracts very similar to file [docWithSecurityScheme.yaml](https://github.com/microsoft/OpenAPI.NET/blob/main/test/Microsoft.OpenApi.Tests/Models/Samples/docWithSecurityScheme.yaml) to highlight the difference between security/no security/empty security. At the same time this introduces duplication into the contracts and the unit tests.

Are you okay with this duplication, or shall I remove that by taking out duplicate content from the contracts?

Reply:
the duplication is fine, thanks for checking